feg/gateway/services/testcore/hss/servicers/auth.go

```diff
@@ -52,18 +52,29 @@ const (
 //   subscriber: The subscriber data for the subscriber we want to generate auth vectors for
 //   plmn: 24 bit network identifier
 //   authSqnInd: the IND of the current vector being generated
-// Returns: The E-UTRAN vectors and the next value to set the subscriber's LteAuthNextSeq to (or an error).
-func GenerateLteAuthVectors(numVectors uint32, milenage *crypto.MilenageCipher, subscriber *protos.SubscriberData, plmn, lteAuthOp []byte, authSqnInd uint64) ([]*crypto.EutranVector, uint64, error) {
-	var vectors = make([]*crypto.EutranVector, 0, numVectors)
+// Returns: The E-UTRAN vectors, UTRAN vectors, the next value to set the subscriber's LteAuthNextSeq to (or an error)
+func GenerateLteAuthVectors(
+	numEutranVectors uint32,
+	numUtranVectors uint32,
+	milenage *crypto.MilenageCipher,
+	subscriber *protos.SubscriberData,
+	plmn,
+	lteAuthOp []byte,
+	authSqnInd uint64) ([]*crypto.EutranVector, []*crypto.UtranVector, uint64, error) {
+
+	if numEutranVectors > 5 {
+		numEutranVectors = 5
```

Comment:
Not important but maybe it would make sense to define a constant on the top of the file 
`maxEutranVectors`
`maxUtranVectors`